Arrays_and_Cycles/js-arrays-9.js

```diff
@@ -0,0 +1,11 @@
+// Реализовать цикл, который выводит в консоль простые числа
+function logPrimeNumbersFromTo(fromNumber, toNumber) {
```

Comment:
Works not fully correct. When I passed as number negatives, 0 or 1 as `fromNumber` I got it as a prime. But it is not prime.